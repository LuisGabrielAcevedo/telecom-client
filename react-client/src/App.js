import React from "react";
import "./App.scss";
import CurrentWeather from "./components/CurrentWeather";
import CitiesWeather from "./components/CitiesWeather";

function App() {
  return (
    <div>
      <CurrentWeather />
      <CitiesWeather />
    </div>
  );
}

export default App;
