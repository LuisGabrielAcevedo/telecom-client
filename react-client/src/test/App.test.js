import React from "react";
import "@testing-library/jest-dom";
import { shallow } from "enzyme";
import App from "../App";

describe("App Component tests", () => {
  it("should init the component", () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
