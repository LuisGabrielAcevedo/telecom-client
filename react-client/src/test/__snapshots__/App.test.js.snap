// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component tests should init the component 1`] = `
<div>
  <CurrentWeather />
  <CitiesWeather />
</div>
`;
